action.yml

```diff
@@ -1,29 +1,31 @@
 name: 'Interface exporter action'
-description: 'An action to generate npm packages based on Solidity contract interfaces'
+description: 'An action to generate npm packages based on solidity interfaces and contracts'
 author: 'Wonderland'
+
 inputs:
-  out_dir:
-    description: 'path to the `out` folder containing the compiled contracts'
-    required: true
-  interfaces_dir:
-    description: 'path to the `interfaces` folder containing the contract interfaces'
-    default: 'src/interfaces'
-    required: false
-  typing_type:
-    description: 'interface compatibility: abi, contracts'
-    required: true
   package_name:
-    description: 'name of the package to be publish'
-    required: true
-  destination_dir:
-    description: 'path to the directory where your package will be exported'
+    description: 'name of the package to be published'
     required: true
-  contracts_dir:
-    description: 'path to the directory where the contracts are located'
+  out:
+    description: 'path to the `out` folder containing the compiled contracts'
+    required: false
+    default: 'out'
+  interfaces:
+    description: 'path to the interfaces directory'
+    required: false
+    default: 'interfaces'
+  contracts:
+    description: 'path to the contracts directory'
+    required: false
+    default: 'contracts'
+  export_type:
+    description: '`abi` for exporting only the interfaces and their ABIs, `contracts` for exporting the contracts and their ABIs as well'
     required: false
+    default: 'interfaces'
+
 outputs:
   passed:
-    description: 'boolean describing if the test passed or not'
+    description: 'boolean describing if the tests passed or not'
```

Comment:
We could choose a more precise name of the output. How about `published` or `exported`? Description: a boolean indicating whether the export has succeeded or not.

Reply:
There is also a reference to `passed` in `main.ts`.

Reply:
I like `exported` 👍